zk/README.md

```diff
@@ -25,16 +25,54 @@ As of version 3.5, Zookeeper has a `4lw.commands.whitelist` parameter (see [Zook
 
 To configure this check for an Agent running on a host:
 
-1. Edit the `zk.d/conf.yaml` file, in the `conf.d/` folder at the root of your [Agent's configuration directory][4] to start collecting your Zookeeper [metrics](#metric-collection) and [logs](#log-collection).
+1. Edit the `zk.d/conf.yaml` file, in the `conf.d/` folder at the root of your [Agent's configuration directory][4] to start collecting your ZooKeeper [metrics](#metric-collection) and [logs](#log-collection).
    See the [sample zk.d/conf.yaml][5] for all available configuration options.
 
 2. [Restart the Agent][6].
 
+#### Enabling SSL
+
+ZooKeeper 3.5 introduced the ability to use SSL authentication. For information about setting up SSL with ZooKeeper, see the [ZooKeeper SSL User Guide][13]. 
+
+After you have ZooKeeper set up with SSL, you can also configure the Datadog Agent to connect to ZooKeeper using SSL. If you already have authentication set up using JKS files, follow the steps below to convert them to PEM files for TLS/SSL configuration.
+
+The following example commands assume that your JKS `truststore` and `keystore` files are called:
+
+- `server_truststore.jks`
+- `server_keystore.jks` 
+- `client_truststore.jks`
+- `client_keystore.jks`
+
+The instructions also assume that both sides' `keystore` and `truststore` files have each other's certificates with aliases `server_cert` and `client_cert`, meaning that a Java ZooKeeper client can already connect to a ZooKeeper server.
+If your private key has a password, make sure this password is included in the `config.yaml` file for config option `tls_private_key_password`.
+
+To convert the JKS files to PEM files:
+
+1. Get the `ca_cert.pem` file from `client_truststore.jks`, since the client's truststore contains the certificate of the server that is trustable:
+    ```
+    keytool -exportcert -file ca_cert.pem -keystore client_truststore.jks -alias server-cert -rfc
+    ```
+   
+2. Get the `cert.pem` file from `server_truststore.jks`, since `cert.pem`:
```

Comment:
... either delete ", since cert.pem" or we are missing the end of the sentence.

Reply:
Oops good catch, will add to sentence.